### PR TITLE
Automated generation of IDE profile files from tornado command

### PR DESCRIPTION
**Pull request by stratika**
#### Problem description

As mentioned in #582, the IDE xml configuration files are being tracked by the vcs system, as they are part of the repository.

#### Description

To address this issue, this PR provides template xml files which are stored in the repository.

At first, a developer can build TornadoVM in console (manually, or via `tornadovm-installer`). 

Once it is built, developers can use the `tornado` command to generate custom IDE files that can be used by them to build and run TornadoVM from the IntelliJ IDE. This is the command:

```bash
tornado --ideinit
```

The command will use the runtime information from the environment variables, in order to configure the template xml files with information such as: `TORNADO_SDK`, `JAVA_HOME`, `BACKENDS`, etc. The generated files are stored in the `.build` directory.

And then if developers open the IDE, they will be able to build and run the project from the IDE.

This PR also updates the relevant documentation page.

I tested the branch in Ubuntu 23.10, macOS 15.0.1 and Windows 11

**Note:** If developers run the command multiple times, the `tornado` script deletes the files and generates new files.

#### Backend/s tested

Mark the backends affected by this PR.

- [x] OpenCL
- [x] PTX
- [x] SPIRV

No backends are affected. This PR does not touch the compiler. However, I tried all configurations to see that the command follows the correct configurations based on the TornadoVM built instances.

#### OS tested

Mark the OS where this PR is tested.

- [x] Linux
- [x] OSx
- [x] Windows

#### Did you check on FPGAs?

If it is applicable, check your changes on FPGAs.

- [ ] Yes
- [x] No

#### How to test the new patch?

Follow the [instructions](https://github.com/stratika/TornadoVM/blob/feat/idea-xml-generation/docs/source/ide-integration.rst).

----------------------------------------------------------------------------

### CODE REVIEW

File: tornado-assembly/src/bin/idea_xml_utils.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+import os
+import subprocess
+import sys
+import xml.etree.ElementTree as ET
+from pathlib import Path
+
+def generate_backend_profiles_as_xml_entries(BACKEND_PROFILES):
+    xml_entries = []
+    for backend in BACKEND_PROFILES:
```
Comment (jjfumero) [inline tornado-assembly/src/bin/idea_xml_utils.py:28]: I would be good to print in STDOUT the files names/path being generated.

**Comment by jjfumero** [comment]
Ok, my fault, I did not have the Python plugin installed for the instance I was running.  The build works

**Review by jjfumero — APPROVED** [review]

**Comment by stratika** [comment]
I addressed the comments, and tested the changes in Linux, macOS, Windows.

**Review by mairooni — APPROVED** [review]
LGTM

**Pull request merged**